.github/workflows/PROD-all-webs-and-studios.yaml

```diff
@@ -0,0 +1,204 @@
+name: PROD - Deploy all webs and studios
+on:
+  workflow_dispatch:
+permissions:
+  id-token: write
+jobs:
+  update-radix-config:
+    runs-on: ubuntu-latest
+    env:
+      SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL }}
+    steps:
+      - name: Checkout internal 🛎️
+        id: checkout-internal
+        uses: actions/checkout@v3
+        with:
+          repository: 'equinor/energyvision-internal'
+          ref: main
+          token: ${{ secrets.PADMS_PAT }}
+      - name: Promote preprod tag to prod 🗒️
+        if: github.ref == 'refs/heads/main'
+        shell: bash
+        id: modify-radix
+        run: |
+          # Install pre-requisite
+          python3 -m venv venv
+          source venv/bin/activate
+          python -m pip install ruamel.yaml
+          python ci/promoteAllToProduction.py web
+          python ci/promoteAllToProduction.py studio
+          git config --global user.name 'github'
+          git config --global user.email 'padms@users.noreply.github.com'
+          git remote set-url origin https://x-access-token:${{ secrets.PADMS_PAT }}@github.com/equinor/energyvision-internal
+          if [[ `git status --porcelain` ]]
+          then
+            git commit -am "🚀 Deploy all web sites and studios to production"
+            git pull --rebase origin main
+            git push origin HEAD:main
+          fi
+      - uses: act10ns/slack@v2
+        with:
+          status: ${{ job.status }}
+          steps: ${{ toJson(steps) }}
+        if: failure()
+
+  deploy-webs:
+    needs: update-radix-config
+    runs-on: ubuntu-latest
+    environment:
+      name: production
+      url: https://equinor.com
+    env:
+      SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL }}
+      DOCKER_BUILDKIT: 1
+    steps:
+      - name: Checkout 🛎️
+        uses: actions/checkout@v3
+
+      - name: Get token 🔑
+        id: get-token
+        uses: ./.github/workflows/get-auth-token/
+
+      - name: Deploy to Radix backup 🚀
```

Comment:
Deploy to backup can be removed. There is a clean up task to remove app from backup cluster